instrumentation/opentelemetry-instrumentation-sqlalchemy/tests/test_sqlcommenter.py

```diff
@@ -46,8 +49,10 @@ def test_sqlcommenter_enabled(self):
         SQLAlchemyInstrumentor().instrument(
             engine=engine,
             tracer_provider=self.tracer_provider,
-            enable_commenter=True,
```

Comment:
Might be nice to add a test to validate `enable_commenter` when no explicit engine are passed.